deepdish/io/hdf5io.py

```diff
@@ -10,20 +10,25 @@
 except ImportError:
     _pandas = False
 
+try:
+    from types import SimpleNamespace as sns
```

Comment:
Is the `as sns` a common practice? Otherwise it might improve readability to simply import it as `SimpleNamepsace`. As it is now `sns` and `_sns` are also very similar.